libfastpm/solver.c

```diff
@@ -117,6 +117,7 @@ void
 fastpm_solver_setup_lpt(FastPMSolver * fastpm,
         enum FastPMSpecies species,
         FastPMFloat * delta_k_ic,
+        FastPMFuncK * growth_rate_k_ic,
```

Comment:
Shall use a name that indicates it is a FuncK, e.g. growth_rate_func_k_ic. Unfortunately the existing code already took the unannotated name for density fields.